packages/mobile/types/atoms.d.ts

```diff
@@ -0,0 +1,29 @@
+import { ReactNode } from "react";
+import { TextProps } from "react-native";
+import colors from "./../../shared/colors";
+
+type TailwindColorKey = keyof typeof colors.light & keyof typeof colors.dark;
+type colorScheme = "light" | "dark";
+
+interface AppTextOwnProps {
```

Comment:
not sure why you need to enforce the values, you could specific the generic values expected that way it's not limited

example size: number